### DISCUSSION TITLE
Support temporal extensions

### DISCUSSION THREAD

**Discussion by peijenlin** [supabase/supabase#20374] [Feature Requests]
Is it possible to support temporal extensions for the database? https://wiki.postgresql.org/wiki/Temporal_Extensions

**Comment by awalias** [comment]
any new extension additions will be made via https://database.dev/ would be cool to see these added there!

**Reply by marktipo** [reply]
So someone needs to add the temporal_tables extension to https://database.dev then? I'm a bit unclear how to browse the currently available extension on the website. 

Is there a plan for someone to upload temporal_tables extension to this repository? If not, is this something @paolochiodi or @mscottx88 from https://github.com/nearform/temporal_tables would consider looking into please? Thanks!
